corehq/apps/userreports/models.py

```diff
@@ -347,6 +347,7 @@ def all(cls):
 
 
 CUSTOM_PREFIX = 'custom-'
+CUSTOM_REPORT_PREFIX = 'customreport-'
```

Comment:
these two constants seem confusing. can we change the first one to `static-` if that makes logical sense and also maybe add docs somewhere?